src/Workspaces/Core/Portable/Shared/Extensions/SemanticModelExtensions.cs

```diff
@@ -194,17 +194,30 @@ private static ISymbol MapSymbol(ISymbol symbol, ITypeSymbol type)
                 declaredSymbol = null;
                 allSymbols = overriddenSymbol is null ? ImmutableArray<ISymbol>.Empty : ImmutableArray.Create(overriddenSymbol);
             }
+            else if (syntaxFacts.IdentifiesLambda(token))
```

Comment:
I'm concerned here as well. GetSemanticInfo is used by several features (including Rename).  It's unclear ot me what the fallout of this would be.  I think i'd prefer that QuickInfo itself just have a fallback path for lambdas (since that's the scenario we're trying to fix up).

Reply:
Thanks for the clarification.
I'll push a fix shortly. That also resolved the F1 regression on => so I'm happy :-)

---
In reply to: [162739605](https://github.com/dotnet/roslyn/pull/23970#discussion_r162739605) [](ancestors = 162739605)